docker/docker-compose.yml

```diff
@@ -269,6 +269,34 @@ services:
       - PLANET_PORT
       - POSTS_URL
 
+  # docusaurus app
+  docs:
+    container_name: 'docs'
+    build:
+      context: ../src/docs
+      dockerfile: Dockerfile
+    environment:
+      - NODE_ENV
+      - DOCS_PORT
+      - DOCS_URL
+    depends_on:
+      - traefik
+    labels:
```

Comment:
If we do this with nginx, we can drop all this, and have `docs` depend only on `nginx`.